app/containers/HomeAddAddress/index.js

```diff
@@ -237,24 +240,46 @@ class AddAddress extends Component<Props> {
   importAddress = () => {
     const {
       activeTab,
-      seed,
+      seeds,
       passPhrase,
       pkh,
       username,
       activationCode
     } = this.state;
+    const input = seeds.toString()
+    const words = input.replace(/["\s]/g, "")
+    const inputVal = words.replace(/,/g, " ")
     this.props.importAddress(
       activeTab,
-      seed,
+      inputVal,
       pkh,
       activationCode,
       username,
       passPhrase
     );
   };
 
+  seedPhraseConvert = (inputValue) => {
+    const words = inputValue.replace(/["\s]/g, "")
+    const seedString = words.replace(/,/g, " ")
+    const seedWords = seedString.split(/\s+/)
+    this.setState({ seeds: seedWords })
+  }
+
+  onChangeInput = (val) => {
+    if(val.length > 15) {
+      this.seedPhraseConvert(val)
```

Comment:
The seedPharseConvert function has to a return value as the Array type.
Please remove `return`